srcpkgs/exfatprogs/template

```diff
@@ -1,18 +1,18 @@
 # Template file for 'exfatprogs'
 pkgname=exfatprogs
-version=1.1.3
-revision=3
+version=1.2.0
+revision=1
 build_style=gnu-configure
 short_desc="Tools to create, check and label exFAT filesystems"
 maintainer="k4leg <python.bogdan@gmail.com>"
-license="GPL-2.0-or-later"
+license="GPL-2.0-only"
```

Comment:
I don't think this change is correct: https://github.com/exfatprogs/exfatprogs/search?q=GPL

Reply:
I don't remember where I got that from but it's definitely wrong.